core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/upgrade/ccm/CcmRemoveAutoSshHandler.java

```diff
@@ -0,0 +1,45 @@
+package com.sequenceiq.cloudbreak.reactor.handler.cluster.upgrade.ccm;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.ccm.upgrade.CcmUpgradeService;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmRemoveAutoSshRequest;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmRemoveAutoSshResult;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.upgrade.ccm.CcmUpgradeFailed;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.EventHandler;
+
+import reactor.bus.Event;
+import reactor.bus.EventBus;
+
+@Component
+public class CcmRemoveAutoSshHandler implements EventHandler<CcmRemoveAutoSshRequest> {
```

Comment:
Instead of implementing the plain `EventHandler` (that is `@Deprecated` anyway), I suggest subclassing `ExceptionCatcherEventHandler`. It will take care of exception capture and event sending as well, so the `EventBus` dependency will not be even needed.

The same applies to all the handlers in this PR.

Reply:
ok